ab_tool/controllers.py

```diff
@@ -61,10 +61,12 @@ def validate_name(name):
     return name
 
 
-def validate_weighting(weight):
-    if not 0 <= weight <=100:
+def validate_weighting(weighting):
+    """ Track weights need to be an integer between 1 and 100 """
+    weighting = int(weighting)
```

Comment:
Where was this an issue?  It looked to me like the edit and create experiment forms have the weighting field as one where the min and max are 0 and 100 and the "step" value is 1.  Was it possible to manually specify half values?  Casting to an int would only fix that case.  I believe if any other string value (empty or otherwise) were entered casting to an int would result in a ValueError exception.  Just curious.